api/db/migrations/20200528170928_add_certif_partner_acquisition_success_value.js

```diff
@@ -0,0 +1,20 @@
+const TABLE_NAME = 'certification-partner-acquisitions';
```

Comment:
Un partenaire de certification != une certification de partenaire.
```suggestion
const TABLE_NAME = 'partner-certification-acquisitions';
```

Reply:
Ca marche, c'est fait (et on supprime le 'acquisition')

```suggestion
const TABLE_NAME = 'partner-certifications';
```

Reply:
Ah intéressant pour le `acquisitions`, je me suis posé la question. Un peu comme pour les badges, le mot `certification` peut désigner le diplôme individuel décerné à un candidat donné (en y réfléchissant, est-ce que le terme correct ne serait pas `certificat` pour ce cas ?), mais aussi le processus de certification lui-même (les critères d'obtention, l'organisme certificateur… ex. "la certification ISO 9001". De ce point de vue, garder `acquisitions` me paraissait éventuellement intéressant parce que (comme pour les badges, encore) ça laissait la place de `partner-certifications` pour le cas où on aurait besoin de parler de la définition de la certification elle-même.
Après, c'est vrai que dans Pix on parle de `certification` tout court (enfin, `certification-courses`…) pour le certificat.
Tout ça pour dire 👍  😄